src/components/TShark/TShark.tsx

```diff
@@ -31,17 +32,21 @@ const TShark = () => {
     // The following component constants are used for rendering the TShark user guide.
     const title = "TShark";
     const description =
-        "TShark is a tool used to capture network traffic and write it to a capture file. The captured traffic can be viewed directly by opening the file, otherwise the Reader option can be used to output the file contents to the output box."; // Description of the component.
+        "TShark is a tool used to capture network traffic and write it to a capture file. The captured traffic can be viewed directly by opening the file, otherwise the Reader option can be used to output the file contents to the output box. The Analyzer option provides traffic analysis options extracting protocol statistics, TCP conversations, and HTTP request data."; // Description of the component.
     const steps =
-        "Step 1: Select Sniffer mode.\n" +
-        "Step 2: Specify the interface to capture traffic from such as eth0.\n" +
-        "Step 3: Specify the output file path (should end with a pcap file).\n" +
-        "Step 4: Input the amount of seconds that TShark should capture traffic for (optional).\n" +
-        "Step 5: Input a packet count to determine the total number of packets to scan before stopping (optional).\n" +
-        "Step 6: Input a traffic filter to filter for specific traffic such as TCP (optional).\n" +
-        "Step 7: Click start TShark to sniff traffic.\n" +
-        "Step 8: Switch to Reader mode.\n" +
-        "Step 9: Provide the file path for the capture file and click start TShark.";
+        "Step 01: Select Sniffer mode.\n" +
+        "Step 02: Specify the interface to capture traffic from such as eth0.\n" +
+        "Step 03: Specify the output file path (should end with a pcap file).\n" +
+        "Step 04: Input the amount of seconds that TShark should capture traffic for (optional).\n" +
+        "Step 05: Input a packet count to determine the total number of packets to scan before stopping (optional).\n" +
+        "Step 06: Input a traffic filter to filter for specific traffic such as TCP (optional).\n" +
+        "Step 07: Click start TShark to sniff traffic.\n" +
+        "Step 08: Switch to Reader mode.\n" +
+        "Step 09: Provide the file path for the capture file and click start TShark.\n" +
+        "Step 10: Switch to Analyzer mode.\n" +
+        "Step 11: Provide the file path for the capture file.\n" +
+        "Step 12: Seletc the Analyzer type and click start TShark";
```

Comment:
Typo in select. Change analyser to analysis to match field. 
```suggestion
        "Step 12: Select the analysis type and click start TShark";
```

Reply:
Hi @s223915833, you missed this change.